### PR TITLE
Include process names in message before installing package.

### LINKED ISSUE #51: Incorrect wording
I have a single instance of the app running, and with this the warning text is a bit off:
image

Perhaps we should check how many instance of the app is actually running (by interrogating running processes?), and adjust the wording appropriately.

Also instead of "these applications" we should say "the application".

### PR DESCRIPTION
**Pull request by maraf**
Fixes #51, an alternative to #52.

image

### CODE REVIEW

File: src/PackageManager.UI/Views/MainWindow.xaml.cs [hunk 1]
```diff
@@ -51,10 +51,11 @@ private Task<bool> OnBeforeChange()
             var context = processes.PrepareContextForProcessesKillBeforeChange();
             if (context.IsExecutable)
             {
+                var processNames = string.Join(" and ", context.ProcessNames);
+
                 bool result = navigator.Confirm(
                     "Plugin Manager",
-                    "Plugin Manager is going to write to files that are holded by other executables. " + Environment.NewLine +
-                    "Do you want to kill all instances of these applications?"
+                    $"Plugin Manager will be writing to files that are currently in use.\r\n\r\nDo you want to stop {processNames}?"
```
Comment (mast-eu) [inline src/PackageManager.UI/Views/MainWindow.xaml.cs:58]:
```suggestion
                    $"Plugin Manager will be writing to files that are currently in use.\r\n\r\nDo you want to stop all instances of {processNames}?"

```

This is really just a detail and I have no strong feelings about it. At the end I accept both forms, with and without this suggestion.

**Review by mast-eu — APPROVED** [review]
I have just one comment, but nothing relevant. For me, this is ok.

Would render #52 obsolete.

File: src/PackageManager.UI/Views/MainWindow.xaml.cs (hunk 1 above)
Comment (maraf) [reply src/PackageManager.UI/Views/MainWindow.xaml.cs:58]: I have no option. I'm not good at user texts, as I tend to put there too much information.
Think it's @RussKie's call 😎

Comment (RussKie) [reply src/PackageManager.UI/Views/MainWindow.xaml.cs:58]: I suppose if there's only a single instance of an app, than it is the first message. Else the second ;))

**Review by RussKie — COMMENTED** [review]
I also suggest changing the messagebox's icon to either question or warning.

LGTM

File: src/PackageManager.UI/Views/MainWindow.xaml.cs (hunk 1 above)
Comment (maraf) [reply src/PackageManager.UI/Views/MainWindow.xaml.cs:60]: No problem 😎

**Comment by maraf** [comment]
Message now contains or don't information about instances based on process count to kill.

**Review by RussKie — APPROVED** [review]

**Pull request merged**